src/components/calculator/Calculate.js

```diff
@@ -0,0 +1,153 @@
+            </button>
+          </div>
+          <div className={classes.row}>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              7
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              8
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              9
+            </button>
+            <div
+              className={`${classes.btn} ${classes.btn_orange}`}
+              onClick={this.clickHandler}
+            >
+              x
+            </div>
+          </div>
+          <div className={classes.row}>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              4
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              5
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              6
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_orange}`}
+              onClick={this.clickHandler}
+            >
+              -
+            </button>
+          </div>
+          <div className={classes.row}>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              1
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              2
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              3
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_orange}`}
+              onClick={this.clickHandler}
+            >
+              ÷
+            </button>
+          </div>
+          <div className={classes.row}>
+            <button
+              className={`${classes.btn} ${classes.btn_0} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              0
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_grey}`}
+              onClick={this.clickHandler}
+            >
+              .
+            </button>
+            <button
+              className={`${classes.btn} ${classes.btn_orange}`}
+              onClick={this.clickHandler}
+            >
+              =
```

Comment:
- [ ]  Please know that you should be using the principle DRY here so you don't repeat yourself much, I recommend that you create a Button component and use what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.